### PR TITLE
Documentation fixes after 1.2 release

### PR DESCRIPTION
**Pull request by SandPod**
Couple of documentation fixes after the release.

Closes: https://github.com/serverpod/serverpod/issues/1772
Closes: https://github.com/serverpod/serverpod/issues/1762
Closes: https://github.com/serverpod/serverpod/issues/1773

### CODE REVIEW

File: docs/12-upgrading/01-upgrade-to-one-point-two.md [hunk 1]
```diff
@@ -231,17 +231,22 @@ $ serverpod create-migration
...
-The repair migration system will create a repair migration that makes your live database schema match the newly created migration. Navigate to your project's `server` package directory and run the `create-repair-migration` command.
+The repair migration system will create a repair migration that makes your live database schema match the newly created migration. To enable the command to fetch your live database schema it requires a running server. Navigate to your project's `server` package directory and start the server, then run the `create-repair-migration` command.
 
 ```bash
+$ dart run bin/main.dart
 $ serverpod create-repair-migration
 ```
 
+:::info
+When starting the server, warnings will be displayed about the database schema not matching the target database schema. These warnings are expected and can safely be ignored.
```
Comment (Isakdl) [inline docs/12-upgrading/01-upgrade-to-one-point-two.md:242]: Suggestion: add "when creating a repair migration."

File: docs/12-upgrading/01-upgrade-to-one-point-two.md [hunk 2]
```diff
@@ -231,17 +231,22 @@ $ serverpod create-migration
...
+
 Use the `--mode` option to specify the database source to use. By default, the repair migration system connects to your `development` database using the information specified in your Serverpod config.
 
 4. Apply the repair migration to your database.
 
-Apply the repair migration to your database using the `--apply-repair-migration` flag when starting the server.
+To apply the repair migration to your database, restart the server using the `--apply-repair-migration` flag.
 
 ```bash
 $ dart run bin/main.dart --apply-repair-migration
```
Comment (Isakdl) [inline docs/12-upgrading/01-upgrade-to-one-point-two.md:252]: Suggestion: add the `maintenance` flag here

**Review by Isakdl — APPROVED** [review]
LGTM! With two small suggestions.

File: docs/12-upgrading/01-upgrade-to-one-point-two.md (hunk 2 above)
Comment (SandPod) [reply docs/12-upgrading/01-upgrade-to-one-point-two.md:252]: My thought was that once you start the server you are ready to test your app. Therefore I didn't add the maintenance mode flag to the instructions.

**Pull request merged**